src/main/java/com/google/devtools/build/lib/collect/nestedset/NestedSet.java

```diff
@@ -368,7 +365,6 @@ private static String childrenToString(Object children) {
         try {
           return Arrays.toString(Futures.getDone(future));
         } catch (ExecutionException e) {
-          logger.atSevere().withCause(e).log("Error getting %s", future);
```

Comment:
this entire else-block (and the dep on flogger) was added in response to an internal crash. the flogger dep is overkill, yes, but can you please still have some sort of log line (using java.util.logging.Logger, i guess) that logs the exception payload? thanks!

Reply:
Done.